.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,11 @@
+---
+name: "❓ Questions and discussions"
+about: You have a question about Expo or want to discuss some aspects of Expo.
+---
+
+<!--
+  If you have a question about Expo or want to discuss about related aspects, consider posting it
+  on our forums at https://forums.expo.io/ or Slack channel at https://slack.expo.io. 🙂 
```

Comment:
Can you update this to mention just the forums for the general public?

Reply:
Sure, updated here https://github.com/expo/expo/commit/bebc9e0668f466e76452589d75352cc62cd67997